### PR TITLE
Significant refactoring 

### PR DESCRIPTION
**Pull request by deadlybutter**
Basically rewrote the entire thing, still missing a good chunk but it's probably good to start reviewing now rather than later

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,339 +1,21 @@
...
-
-This General Public License does not permit incorporating your program into
-proprietary programs.  If your program is a subroutine library, you may
-consider it more useful to permit linking proprietary applications with the
-library.  If this is what you want to do, use the GNU Lesser General
-Public License instead of this License.
\ No newline at end of file
+The MIT License (MIT)
+
+Copyright (c) 2014 DoSomething.org
```
Comment (DFurnes) [inline LICENSE]: 2015! ;)

Comment (deadlybutter) [inline LICENSE]: wait I thought we only changed the time last sunday, not the year?!

File: app.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+app.use('/module/campaigns', router_campaigns);
+this.module_campaigns = require(__dirname + '/modules/campaigns.js')(this, router_campaigns);
+
+app.get('/', function(req, res){
+  res.redirect('/module/static');
+});
+
+//Setup HTTP & data fetchers
+var PORT = process.env.PORT || 3000;
+http.listen(PORT, function(){
```
Comment (DFurnes) [inline app.js]: This could be simplified to `app.listen(PORT);`, which would remove the need for the extra `http` dependency.

**Comment by DFurnes** [comment]
Looks pretty good! :ship:

File: app.js (hunk 2 above)
Comment (deadlybutter) [inline app.js]: HTTP is required for initializing socketio :/

Comment (deadlybutter) [inline app.js]: Relevant line: https://github.com/DoSomething/DS-Realtime-Feed/blob/v2/services/socket.js#L9

Comment (DFurnes) [inline app.js]: Ahhhh, bummer. Gotcha.

**Pull request merged**